### PR TITLE
bagger: bump version to 0.4

### PR DESCRIPTION
**Pull request by artemyevav**
Bagger 0.4 was released recently, this is the repo update to use new bagger binary.

### CODE REVIEW

File: app-admin/bagger-tools/bagger-tools-0.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (monsieurp) [inline app-admin/bagger-tools/bagger-tools-0.4.ebuild:4]:
```suggestion
EAPI=7
```

File: app-admin/bagger-tools/bagger-tools-0.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+if [[ ${PV} == 9999 ]]
+then
+	inherit git-r3
+	KEYWORDS=""
+	EGIT_REPO_URI="https://github.com/adjust/bagger.git"
+else
+	inherit vcs-snapshot
+	SRC_URI="https://github.com/adjust/${PN}/archive/v${PVR}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
```
Comment (monsieurp) [inline app-admin/bagger-tools/bagger-tools-0.4.ebuild:21]:
```suggestion
      KEYWORDS="amd64"
```
Our software releases don't need to be keyworded I think. ;)

File: app-admin/bagger-tools/files/schaufel_listener-0.4.initd [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+		LISTENER_OPTS+=" -g ${LISTENER_BROKER_GROUP}"
+	[ -n "${LISTENER_BROKER}" ] && \
+		LISTENER_OPTS+=" -b ${LISTENER_BROKER}"
+fi
+	[ -n "${INFIX}" ] && \
+		LISTENER_OPTS+=" -i ${INFIX}"
+[ -n "${HOSTNAME_OVERRIDE}" ] && \
+		LISTENER_OPTS+=" -h ${HOSTNAME_OVERRIDE}"
+[ -n "${LISTENER_JOBS}" ] && \
+	LISTENER_OPTS+=" -j ${LISTENER_JOBS}"
```
Comment (monsieurp) [inline app-admin/bagger-tools/files/schaufel_listener-0.4.initd:45]: This entire piece of code can be moved to the conf.d file. Please move it and I'll make some suggestions to improve it then.

File: app-admin/bagger-tools/files/schaufel_listener-0.4.initd [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+pidfile="${LISTENER_PID}"
+
+start_stop_daemon_args="--background \
+	--make-pidfile \
+	--verbose \
+	--chdir ${LISTENER_DIR} \
+	--user ${LISTENER_USER} \
+	--group ${LISTENER_GROUP} \
+	--stdout ${LISTENER_LOG_FILE} \
+	--stderr ${LISTENER_LOG_FILE}"
```
Comment (monsieurp) [inline app-admin/bagger-tools/files/schaufel_listener-0.4.initd:58]:
```suggestion
start_stop_daemon_args="--verbose --user ${LISTENER_USER} --group ${LISTENER_GROUP}"
command_background="true"
directory="${LISTENER_DIR}"
output_log="${LISTENER_LOG_FILE}"
error_log="${LISTENER_LOG_FILE}"

**Review by monsieurp — CHANGES_REQUESTED** [review]

**Comment by artemyevav** [comment]
All these changes look like a `quality of life`. That's out of a scope of this PR, the only thing I need is bumping a version to be able to install 0.4.

**Comment by monsieurp** [comment]
Why are you asking for my review then? :thinking: If the ebuild works and is installable, commit it straight to master and call it a day in this case.

**Review by monsieurp — APPROVED** [review]

File: app-admin/bagger-tools/bagger-tools-0.4.ebuild (hunk 2 above)
Comment (einhverfr) [reply app-admin/bagger-tools/bagger-tools-0.4.ebuild:21]: I think we are keywording btw.  We might want to rethink our policies there though.  I will leave that up to you and David

**Pull request merged**